monty/utils/markdown.py

```diff
@@ -210,11 +209,11 @@ def list(self, text: str, ordered: bool, level: int, start: Any = None) -> str:
         # todo: figure out how this should actually work
         if level != 1:
             return text
-        return text.lstrip("\n") + "\n\n"
+        return text.lstrip("\n") + "\n"
 
     def list_item(self, text: Any, level: int) -> str:
         """Show the list, indented to its proper level."""
-        return "\n" + "\u200b " * (level - 1) * 8 + f"- {text}"
+        return "\u2001" * (level - 1) + f"- {text.rstrip()}\n"
```

Comment:
just noticed this should have some of the same code as block_quote, in accordance with splitting lines for long quotes.

Reply:
This should almost perfectly line up, I don't think it'll get any closer without a monospace font: https://github.com/onerandomusername/monty-python/pull/234/commits/c61f2bc9f4b295b07539cb2425a422a6cafe3499

![image](https://user-images.githubusercontent.com/8530778/228343323-982f11ab-5a89-4c17-a525-bd0a1880d4ee.png)

Reply:
This seems to add a bug in some code blocks

Reply:
I believe I've somewhat fixed this in [`0f3c35f` (#234)](https://github.com/onerandomusername/monty-python/pull/234/commits/0f3c35f9b4dd292bf173cb5a3d4f8ab9f24dd85c)